mlflow/metrics/genai/genai_metric.py

```diff
@@ -229,6 +231,7 @@ def process_example(example):
         name,
         definition,
         grading_prompt,
+        include_input,
```

Comment:
On line 243 in [eval_fn](https://github.com/mlflow/mlflow/pull/10698/files#diff-9899788b9f82a506c9338ef9a5065f61e0484c33321a669ba9facb5139e80e85R243) as args, should we remove inputs as a required field that is passed to the eval_fn?
That also means that in the signature it is optional. Thoughts?